Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM balena/open-balena-base:v4.4.2
 EXPOSE 80
 
 COPY package.json package-lock.json /usr/src/app/
-RUN npm ci --unsafe-perm --production && npm cache clean --force
+RUN npm ci --unsafe-perm --ignore-scripts && npm cache clean --force
```

Comment:
You probably still want the `--production`

Reply:
Oh right, we're not building the project anymore.